### PR TITLE
[stable/metabase] Upgrade metabase to 0.36.3

### PR DESCRIPTION
**Pull request by yachtcaptain23**
**Is this a new chart**
No

**What this PR does / why we need it**
Upgrades metabase from 0.35.3 to 0.36.3. There's new features, bugfixes, and performance enhancements starting from 0.36.0.

**Checklist**
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]

[x] DCO signed
[x] Chart Version bumped
[x] Variables are documented in the README.md
[x] Title of the PR starts with chart name (e.g. [stable/mychartname])

### CODE REVIEW

File: stable/metabase/Chart.yaml [hunk 1]
```diff
@@ -2,8 +2,8 @@ apiVersion: v1
 description: The easy, open source way for everyone in your company to ask questions
   and learn from data.
 name: metabase
-version: 0.12.0
-appVersion: v0.35.3
+version: 0.12.1
+appVersion: v0.36.3
```
Comment (antonu17) [inline stable/metabase/Chart.yaml:6]: Shouldn't patch reset to 0 if minor is bumped?

Comment (pmint93) [reply stable/metabase/Chart.yaml:6]: You mean the appVersion ?
If so then yes, of course there is v0.36.0, v0.36.1, v0.36.2, ... and at the time of this PR v0.36.3 is the latest version.

**Pull request merged**